gcc/rust/util/rust-attributes.cc

```diff
@@ -42,6 +42,7 @@ static const BuiltinAttrDefinition __definitions[]
      {"rustc_builtin_macro", EXPANSION},
      {"path", EXPANSION},
      {"macro_use", NAME_RESOLUTION},
+     {"macro_export", NAME_RESOLUTION},
```

Comment:
Should this be part of this PR?

Reply:
Some tests were failing as we try to expand it when it's not recognized as "builtin".